.travis.yml

```diff
@@ -26,9 +26,6 @@ install:
 script:
   - npm run-script coverage
 
-after_script:
-  - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
```

Comment:
Well funny enough, I'm not the owner of the repo anymore. If I was, I would be able to access the settings of that repo.

![image](https://cloud.githubusercontent.com/assets/11187343/22621289/1c90843a-ead5-11e6-8df9-78a80ec34f0d.png)

Also in coveralls.io, I would have been able to find it there, but not in this case:

![image](https://cloud.githubusercontent.com/assets/11187343/22621285/03e0317e-ead5-11e6-9ecd-c7e5c512ebd3.png)

I'm assuming that by transferring the repo to the org, the admin(s) of alexa-js supposedly own the repo in terms of accessing settings and managing permissions with 3rd party tools, which is why I thought you could manage that.